src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js

```diff
@@ -0,0 +1,132 @@
+    }
+
+    changeCheckboxValue = (value) => {
+        // eslint-disable-next-line camelcase
+        this.setState({ toolbarConfig: { ...this.state.toolbarConfig, show_all: !value }}, this.apply);
+    }
+
+    getCVSSValue = (options) => {
+        const option = options.find(
+            // eslint-disable-next-line camelcase
+            item => item.from === this.state.toolbarConfig.cvss_from && item.to === this.state.toolbarConfig.cvss_to
+        );
+        return option ? option.value : options[0].value;
+    }
+
+    render() {
+        const { showAllCheckbox, downloadReport, totalNumber } = this.props;
+
+        return (
+            <React.Fragment>
+                <Toolbar className="cvetable-toolbar ">
+                    <ToolbarGroup className="filterCVES space-between-toolbar-items">
+                        <ToolbarItem>
+                            <SimpleTableFilter onFilterChange={ (value) => this.changeFilterValue(value) } onButtonClick={ this.apply } />
+                        </ToolbarItem>
+                        <ToolbarItem>
+                            <Form>
+                                <FormGroup label="CVSS Base Score" fieldId="cvssScore">
+                                    <Select
+                                        id="cvssScore"
+                                        onChange={ value => this.changeCVSSValue(value, CVSSOptions) }
+                                        value={ this.getCVSSValue(CVSSOptions) }
+                                    >
+                                        { CVSSOptions.map((option, index) => (
+                                            <SelectOption
+                                                isDisabled={ option.disabled }
+                                                key={ index }
+                                                value={ option.value }
+                                                label={ option.label }
+                                            />
+                                        )) }
+                                    </Select>
+                                </FormGroup>
+                            </Form>
+                        </ToolbarItem>
+                    </ToolbarGroup>
+                    <ToolbarGroup>
+                        <ToolbarItem>
+                            { showAllCheckbox && (
+                                <React.Fragment>
+                                    <br />
+                                    <Checkbox
+                                        label="Hide CVEs that do not affect my inventory"
+                                        isChecked={ !this.state.toolbarConfig.show_all }
+                                        onChange={ state => this.changeCheckboxValue(state) }
+                                        aria-label="controlled checkbox example"
```

Comment:
this should be more specific, unsure if we have a convention around labels... something like `vulnerabilities toolbar hide cves checkbox` might be a bit more useful